ceph/utils.py

```diff
@@ -165,6 +165,33 @@ def volume_cleanup(volume, osp_cred):
                     return 1
 
 
+# Added this function to cleanup stale volumes with status deleting, available, error, unknown, reserved
+def cleanup_ceph_vols(osp_cred, pattern=None, timeout=300):
```

Comment:
I think pattern param is not used, are we deleting the volumes by name pattern?

Reply:
it would be good if we use name pattern, so it would be safe delete.

Reply:
Not using name pattern, to clean volumes which are not attached to any known VM instances